_applications/equoid.adoc

```diff
@@ -0,0 +1,223 @@
+oc create -f https://radanalytics.io/resources.yaml
+oc create -f https://raw.githubusercontent.com/infinispan/infinispan-openshift-templates/master/templates/infinispan-ephemeral.json
+....
++
+These provide the Fabric8 fuse-java image stream for the data-publisher service, the radanalytics.io Oshinko resources for facilitating Spark cluster creation and deployment as well as S2I for the data-handler, and the Infinispan template for the caching microservice, respectively. 
+
+. Launch Artemis
++
+We'll get Artemis up and running, as it is necessary for both the publisher and handler:
++
+....
+oc create -f https://raw.githubusercontent.com/radanalyticsio/equoid-openshift/master/artemis-rc.yaml
+....
+
+. Launch Infinispan Cache
++
+We'll want our caching layer in place before we begin streaming and ingesting. The following will get an Infinispan pod running. `APPLICATION_USER`, `APPLICATION_PASSWORD`, `MANAGEMENT_USER` and `MANAGEMENT_PASSWORD` should be changed to your desired values. For purposes of this example we'll use `radlee` for all our credentials. 
++
+....
+oc new-app --template=infinispan-ephemeral \
+    -l app=datagrid \
+    -p APPLICATION_NAME=datagrid \
+    -p NAMESPACE=`oc project -q` \
+    -p APPLICATION_USER=radlee \
+    -p APPLICATION_PASSWORD=radlee \
+    -p MANAGEMENT_USER=radlee \
+    -p MANAGEMENT_PASSWORD=radlee
+....
++
+
+. Launch Data Publisher
++
+We can next begin publishing our streaming data by starting an instance of the data publisher. `OP_MODE` described later should be set to the mode you wish. Also, `DATA_URL_PRIMARY` should be set to the location of your data set or data potential values file. For purposes of this example we'll use linear processing of a full dataset contained in the `equoid-data-publisher` github repo.  
++
+....
+oc new-app \
+    -l app=publisher \
+    -e OP_MODE=linear \
+    -e DATA_URL_PRIMARY=https://raw.githubusercontent.com/radanalyticsio/equoid-data-publisher/master/data/StockCodesLinear.txt  \
+    --image-stream=`oc project -q`/fuse-java \
+    https://github.com/radanalyticsio/equoid-data-publisher
+....
+
+. Launch Data Handler
++
+We are now ready to start our data handler. Note that `INFINISPAN_HOST` and `INFINISPAN_PORT` need to be set to appropriate values, in the case of the template provided, these are `datagrid-hotrod` and `11222` respectively. In addition, `WINDOW_SECONDS` is the number of second for which you wish to track the top-k items, `SLIDE_SECONDS` represents the number of seconds to slide your data window each iteration, and `BATCH_SECONDS` is the size of samples you wish to acquire from the message stream at a time. Finally, `OP_MODE` needs to be set to the same value as set for the data publisher. In our example we will use linear corresponding with our previous setting for the data publisher, and 20 seconds for the window, slide, and batch sized. 
```

Comment:
nit: "is the number of second" --> "is the number of seconds"

we will use linear corresponding --> we will use "linear" corresponding ?

Reply:
ACK, added to revision, thanks